integration_test/iptables/iptablestest-lab.yaml

```diff
@@ -274,3 +274,56 @@ spec:
   volumes:
   - emptyDir: {}
     name: linkerd-proxy-init-xtables-lock
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-ignores-subnets
+  labels:
+    app: pod-ignores-subnets
+spec:
+  containers:
+  - name: proxy-stub
+    image: ghcr.io/linkerd/iptables-tester:v1
+    env:
+    - name: PORT
+      value: "8080"
+    - name: AM_I_THE_PROXY
+      value: "yes"
+    command: ["go", "run", "/go/test_service/test_service.go"]
+    ports:
+    - name: http
+      containerPort: 8080
+    securityContext:
+      privileged: false
+      runAsUser: 2102
+  - name: other-container
+    image: ghcr.io/linkerd/iptables-tester:v1
+    env:
+    - name: PORT
+      value: "9090"
+    command: ["go", "run", "/go/test_service/test_service.go"]
+    ports:
+    - name: http
+      containerPort: 9090
+  initContainers:
+  - name: linkerd-init
+    image: ghcr.io/linkerd/proxy-init:latest
+    imagePullPolicy: Never
+    args: ["-p", "8080",  "-o", "8080", "-u", "2102", "--subnets-to-ignore", "0.0.0.0/0"]
```

Comment:
I tried ignoring `0.0.0.0/32` instead of anywhere (`0.0.0.0/0`), but the requests are still getting routed to the proxy container. I thought the HTTP server from `test_service.go` is bind to `0.0.0.0` 🤔

Reply:
Hm yeah, it should be bound to `0.0.0.0`. Looking at the code, we don't pass in a host and I think by default it will listen on any address (so 0.0.0.0). We start the listener [here](https://github.com/linkerd/linkerd2-proxy-init/blob/master/integration_test/iptables/test_service/test_service.go#L54). Odd thought, I was under the impression `0.0.0.0/32` would work, but ignoring anywhere also sounds good to me.